react/features/remote-video-menu/components/web/MuteEveryoneElseButton.js

```diff
@@ -0,0 +1,77 @@
+/* @flow */
+
+import React from 'react';
+
+import { createToolbarEvent, sendAnalytics } from '../../../analytics';
+import { openDialog } from '../../../base/dialog';
+import { translate } from '../../../base/i18n';
+import { IconMicDisabled } from '../../../base/icons';
+import { connect } from '../../../base/redux';
+
+import AbstractMuteButton, {
+    _mapStateToProps,
+    type Props
+} from '../AbstractMuteButton';
+
+import RemoteVideoMenuButton from './RemoteVideoMenuButton';
+import MuteEveryoneDialog from '../../../remote-video-menu/components/web/MuteEveryoneDialog';
+
+/**
+ * Implements a React {@link Component} which displays a button for audio muting
+ * a participant in the conference.
+ *
+ * NOTE: At the time of writing this is a button that doesn't use the
+ * {@code AbstractButton} base component, but is inherited from the same
+ * super class ({@code AbstractMuteButton} that extends {@code AbstractButton})
+ * for the sake of code sharing between web and mobile. Once web uses the
+ * {@code AbstractButton} base component, this can be fully removed.
+ */
+class MuteEveryoneElseButton extends AbstractMuteButton {
+    /**
+     * Instantiates a new {@code Component}.
+     *
+     * @inheritdoc
+     */
+    constructor(props: Props) {
+        super(props);
+
+        this._handleClick = this._handleClick.bind(this);
+    }
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        const { participantID, t } = this.props;
+
+        return (
+            <RemoteVideoMenuButton
+                buttonText = { t('videothumbnail.domuteOthers') }
+                displayClass = { 'mutelink' }
+                icon = { IconMicDisabled }
+                id = { `mutelink_${participantID}` }
+                // eslint-disable-next-line react/jsx-handler-names
```

Comment:
Why?

Reply:
So, I took the example from:
react/features/remote-video-menu/components/web/MuteButton.js:60

After some further investigation it seems this is necessary because in AbstractMuteButton the click handler is named without 'on' thus breaking the 'react/jsx-handler-names' rule.

How should I proceed then?

Reply:
I'd say you can leave it, as this is the case already with MuteButton.